airbyte_cdk/cli/airbyte_cdk/_connector.py

```diff
@@ -162,6 +162,13 @@ def test(
         pytest_args.append("--collect-only")
 
     pytest_args.append(str(test_file_path))
+
+    if connector_directory:
+        build_dir = connector_directory / "build" / "test-results"
+        build_dir.mkdir(parents=True, exist_ok=True)
+        junit_xml_path = build_dir / "standard-tests-junit.xml"
+        pytest_args.extend(["--junitxml", str(junit_xml_path)])
```

Comment:
I think we have to assume the connector directory exists, right? I'll apply this change, but let me know if we have an issue of connector_directory sometimes being null.

```suggestion
    build_dir = connector_directory / "build" / "test-results"
    build_dir.mkdir(parents=True, exist_ok=True)
    junit_xml_path = build_dir / "standard-tests-junit.xml"
    pytest_args.extend(["--junitxml", str(junit_xml_path)])
```